vng_api_common/authorizations/models.py

```diff
@@ -54,3 +55,7 @@ class Autorisatie(APIMixin, models.Model):
     max_vertrouwelijkheidaanduiding = VertrouwelijkheidsAanduidingField(
         help_text=_("Maximum level of confidentiality that is allowed")
     )
+
+    def satisfy_vertrouwelijkheid (self, vertrouwelijkheidaanduiding) -> bool:
+        return VertrouwelijkheidsAanduiding.get_choice(self.max_vertrouwelijkheidaanduiding).order >= \
```

Comment:
backslashes are ugly, it's more readable in general to assign both choices to a variable and do the comparison based of that. The extra few lines of code don't matter, in fact, they'll probably make it more readable

Reply:
fixed